### PR TITLE
LELLKI WP-30-EU Outlet 4 sockets EU, 2 USB (With Power Measurment) 

### PR DESCRIPTION
**Pull request by DavidFFerreira**
Add Power Cord 4 sockets EU, 2 USB (With Power Measurment)  LeLLKI W30-EU
Aliexpress:
https://s.click.aliexpress.com/e/_AVhWJo

Picture of the device:
image

Profe of working
image
image

### CODE REVIEW

**Review by DavidFFerreira — COMMENTED** [review]
spaces fixes

**Review by DavidFFerreira — COMMENTED** [review]
update spaces

**Review by DavidFFerreira — COMMENTED** [review]
fix spaces

**Review by DavidFFerreira — COMMENTED** [review]
identation with tab and spaces sucks -_-

**Comment by DavidFFerreira** [comment]
@Koenkk can you release in the next update? When?

File: devices/lellki.js [hunk 1]
```diff
@@ -82,4 +83,31 @@ module.exports = [
...
+            endpoint.saveClusterAttributeKeyValue('seMetering', {divisor: 100, multiplier: 1});
+            device.save();
+        },
+        options: [exposes.options.measurement_poll_interval()],
+        exposes: [e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'),
+            e.switch().withEndpoint('l3'), e.power(), e.current(), e.voltage().withAccess(ea.STATE),
+            e.energy(), exposes.enum('power_outage_memory', ea.STATE_SET, ['on', 'off', 'restore'])
+                .withDescription('Recover state after power outage')],
+        endpoint: (device) => {
+            return {l1: 1, l2: 2, l3: 3};
```
Comment (Koenkk) [inline devices/lellki.js:109]: I see only 3 switches here while the description says it has 4 sockets + 2 usb

Comment (DavidFFerreira) [reply devices/lellki.js:109]: Hi, unfortynally there is no more IOS on the board, soo 4 Sockets, but only 3 sockets work Remotly =)

**Comment by Koenkk** [comment]
thanks!

**Pull request merged**